src/components/Crypto_Main_Chart/MainChart.jsx

```diff
@@ -0,0 +1,114 @@
+import React, { FunctionComponent } from "react";
+import {
+  LineChart,
+  Line,
+  XAxis,
+  YAxis,
+  CartesianGrid,
+  Tooltip,
+  Legend,
+  LabelList
+} from "recharts";
+
+const data = [
```

Comment:
Create a separate folder for data and create this data there. Finally import it here to use